samples/unity-of-bugs/ProjectSettings/ProjectSettings.asset

```diff
@@ -195,7 +195,7 @@ PlayerSettings:
   keepLoadedShadersAlive: 0
   StripUnusedMeshComponents: 1
   VertexChannelCompressionMask: 4054
-  iPhoneSdkVersion: 988
+  iPhoneSdkVersion: 989
```

Comment:
Target changed from iOS device to Simulator.

Reply:
If we change the target from Simulator to Iphone/Ipad we'll not be able to test the app on a Simulator.